Contact form

ojsdude opened the pull request:
- Added the form to the `Still unsure?` section
- Added the howdy form script
- Removed direct support@weareleaf.com mailtos from docs
- Added the ability to add `noflex` as a class on `.alert` elements
- Added form scss
- Also added a small negative margin to the users svg to ensure that there is no gap between that and the footer

ojsdude committed:
Fixed users svg spacing

- Added a small negative margin to the users svg to ensure that there is no gap between that and the footer

ojsdude committed:
Added the contact-us form to the Docs page

- Added the form to the `Still unsure?` section
- Added the howdy form script
- Removed direct support@weareleaf.com mailtos from docs
- Added the ability to add `noflex` as a class on `.alert` elements
- Added form scss

michaelcarter reviewed (commented):
Looks good, just a few styling bits:

<img width="285" alt="screen shot 2017-08-28 at 11 24 11" src="https://user-images.githubusercontent.com/1911741/29769931-62c15956-8be4-11e7-9531-ab459c918d8d.png">

Could we get all the links in the contents to the same size (the same size as paragraph text on the page), and add a "Table Of Contents" heading above please?

<img width="765" alt="screen shot 2017-08-28 at 11 23 36" src="https://user-images.githubusercontent.com/1911741/29769967-89bbcdca-8be4-11e7-85c1-bab9209e85ff.png">

Could you please:
 - Align the text to the left
 - Put the life ring back to the left like it is on the current live site
 - Change the background colour of the contact form inputs to `D1EBFF` (the same shade of blue as the background of the life ring
 - Reduce the padding on the inputs a bit
 - Remove the margin below the contact form, so it fits snugly within the blue info area.

If @chrisannetts says anything that contradicts what I've said here, listen to him, not me. 😆

ojsdude committed:
Changes based on review

ojsdude committed:
Merge branch 'master' into contact-form

ojsdude committed:
Added the additional tracking

- track "Viewed Home Page" when the user views the homepage
- track "Viewed Docs Page" when the user views the docs page (but not a specific topic)
- track "Viewed payment form" when the user lands on the start subscription page
- track "Started subscription" when the user successfully starts a subscription

ojsdude committed:
Merge pull request #7 from weareleaf/additional-tracking

Additional tracking
